query/src/interpreters/interpreter_user_stage_create.rs

```diff
@@ -55,14 +55,7 @@ impl Interpreter for CreateUserStageInterpreter {
         if user_stage.stage_type == StageType::Internal {
             let prefix = format!("stage/{}/", user_stage.stage_name);
             let op = self.ctx.get_storage_operator()?;
-            op.object(&prefix).create().await.or_else(|e| {
-                if e.kind() == std::io::ErrorKind::AlreadyExists {
-                    // if target path already exist, just ignore the error
-                    Ok(())
-                } else {
-                    Err(e)
-                }
-            })?;
+            op.object(&prefix).create().await?
```

Comment:
```suggestion
            op.object(&prefix).create().await?;
```

Reply:
Seems the type of `op.object(&prefix).create().await?` is `()`, those two expressions equal with each other?.

Reply:
Oh, yep, lesson learned.